lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -1667,6 +1671,25 @@ MDagPath PrimUpdaterManager::setupPullParent(const Ufe::Path& pulledPath, VtDict
     return pullParentPath;
 }
 
+void PrimUpdaterManager::recordPullVariantInfo(
+    const Ufe::Path& pulledPath,
+    const MDagPath&  pullParentPath)
+{
+    // Add the pull parent to our pulled prims prefix tree.  Also add the full
+    // configuration of variant set selections for each ancestor, up to the USD
+    // pseudo-root.  Variants on the pulled path itself are ignored, as once
+    // pulled into Maya they cannot be changed.
+    TF_VERIFY(!_pulledPrims.containsDescendantInclusive(pulledPath));
```

Comment:
Don't we already prevent editing a child or parent of an already pull prim? I guess rechecking here is harmless, I just want to understand if this protects against somethings I'm not seeing.